v3/src/models/data/attribute.ts

```diff
@@ -40,14 +40,26 @@ export const kDefaultFormatStr = ".3~f"
 const isDevelopment = () => process.env.NODE_ENV !== "production"
 const isProduction = () => process.env.NODE_ENV === "production"
 
-export type IValueType = string | number | boolean | undefined
+export type IValueType = string | number | boolean | undefined | Date
```

Comment:
I would put the `undefined` last, so:
```suggestion
export type IValueType = string | number | boolean | Date | undefined
```